smugmug.com.custom-domain.json

```diff
@@ -1,21 +1,23 @@
-{  
-   "providerId":"smugmug.com",
-   "providerName":"SmugMug",
-   "serviceId":"custom-domain",
-   "serviceName":"SmugMug Custom Domain",
-   "version": 2,
-   "logoUrl":"https://www.domainconnect.org/wp-content/uploads/2016/02/smugmug-300x80.png",
-   "description":"Enables a domain to work with SmugMug",
-   "records":[
-      {  
+{
+   "providerId": "smugmug.com",
+   "providerName": "SmugMug",
+   "serviceId": "custom-domain",
+   "serviceName": "SmugMug Custom Domain",
+   "version": 3,
+   "logoUrl": "https://cdn.smugmug.com/img/domain-connect/smugmug-logo.png",
+   "description": "Enables a domain to work with SmugMug",
+   "syncRedirectDomain": "smugmug.com, smugmug.net",
+   "records": [
+      {
          "type": "REDIR301",
-         "target": "www.%fqdn%"
+         "target": "www.%domain%",
+         "host": "%domain%."
```

Comment:
Sorry for not reviewing this template any earlier, but this construct does look strange.
If there is no intention to use this template with host variable and only for APEX, why to bother with the `%domain%.` construct where `@` would do exactly the same?
Note that you may face providers not implementing trailing dot the right way as it was quite late addition to the spec.

Reply:
`@` and `%domain%.` behave the same only when there is no host provided for template application.  While we don't intend to provide a host, codifying the behavior in the template reduces the potential for implementation bugs and serves as a documentation of intended behavior.